packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2312,6 +2377,45 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
     }
   }
 
+  /// Mark this render object as having changed a property on its composited
+  /// layer.
+  ///
+  /// Render objects that have a composited layer and are repaint boundaries
```

Comment:
somehow link to isRepaintBoundary from this doc

Reply:
Done